tests/spherical_nsinker.prm

```diff
@@ -0,0 +1,15 @@
+set Dimension = 3
```

Comment:
You don't need that line, right?

Reply:
Yes, it is necessary as the dimension can not be read from an include statement. I don't think we document this anywhere, but this is sadly the case.

Reply:
Oh wow, that's good to know!